src/controllers/hid/hidioreport.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "controllers/controller.h"
+#include "controllers/hid/hiddevice.h"
+#include "util/duration.h"
+
+class HidIoReport {
+  public:
+    HidIoReport(const unsigned char& reportId,
+            hid_device* device,
+            const mixxx::hid::DeviceInfo&& deviceInfo);
+    void sendOutputReport(QByteArray data);
+
+  private:
+    const unsigned char m_reportId;
+    const RuntimeLoggingCategory m_logOutput;
+    hid_device* const
```

Comment:
should be double-const I think?  (const hid_device* const)

Reply:
No, const will not work here, this is the structure where HIDAPI functions store information like the last error message.